plugins/rebalance/rest.go

```diff
@@ -188,34 +197,33 @@ func rebalanceStopHandler(w http.ResponseWriter, r *http.Request) {
 
 	err = txn.Ctx.Set("volname", volname)
 	if err != nil {
-		logger.WithError(err).Error("failed to set volname in transaction context")
+		logger.WithError(err).WithField("key", "volname").Error("failed to set volname in transaction context")
 		restutils.SendHTTPError(ctx, w, http.StatusInternalServerError, err.Error())
 		return
 	}
 
-	rebalinfo.Volname = volname
-	rebalinfo.State = rebalanceapi.Stopped
-	rebalinfo.Cmd = rebalanceapi.CmdStop
+	rebalInfo.Volname = volname
```

Comment:
question: 

do we need to assign ```rebalInfo.Volname = volname``` as Volname already assigned during rebalance start?